python/kfserving/setup.py

```diff
@@ -25,7 +25,7 @@
 
 setuptools.setup(
     name='kfserving',
-    version='0.2.1',
+    version='0.2.4',
```

Comment:
Why jumping to 4? Isn't 2 the right way?

Reply:
Let me explain: you know there are many changes after 0.2.1, then I released new pkg for fairing integration, and then I found the `import` issue running in fairing after intergration then I released new one for fairing CI test pass. And now you refactor the kfserving server, I would like to update the new version again and release new version to include your changes. :-)
I think that's not big deal since we need release 0.3 accordingly if kfserving cut 0.3 tag or version chanegd. Thanks @ellis-bigelow

Reply:
looks sdk is already on 0.2.3, let's use 0.3.0.x for fixes with 0.3 release.